app/assets/javascripts/actions/article_finder_action.js

```diff
@@ -218,6 +220,7 @@ export const fetchKeywordResults = (keyword, home_wiki, offset = 0, continueResu
     return data.query.search;
   })
   .then((articles) => {
+    if (home_wiki.project === 'wikidata') fetchWikidataLabels(articles, dispatch);
```

Comment:
Do the `articles` here have a `project` property? If so, this conditional could be replaced with a call to `fetchWikidataLabelsForArticles` (which already handles filtering for wikidata).

Reply:
@ragesoss No, it doesn't have that property from the API call.